src/screens/MediumActivity/MediumActivity.js

```diff
@@ -0,0 +1,78 @@
+import React, {Component} from "react";
+import {
+  View,
+  Text,
+  SafeAreaView,
+  FlatList,
+  Image,
+  Linking,
+  TouchableOpacity,
+} from "react-native";
+import styles from "./styles";
+import config from "../../config/config";
+
+class MediumActivity extends Component {
+  state = {
+    mdata: [],
+  };
+
+  FetchDataFromRssFeed() {
+    const {mdata} = this.state;
+    var request = new XMLHttpRequest();
+    request.onreadystatechange = () => {
+      if (request.readyState == 4 && request.status == 200) {
+        var Obj = JSON.parse(request.responseText);
+        this.setState({mdata: Obj.items});
+      }
+    };
+    request.open("GET", config.MEDIUM_API, true);
```

Comment:
Cant we do something like this here,

import { MEDIUM_API } from "../../config/config";
...
request.open("GET", MEDIUM_API, true);

I think we do not need to import all the things in the config here. This should increase the performance also.

Reply:
Done!